src/api/public/apidocs-new/OBS-v2.10.50.yaml

```diff
@@ -311,6 +311,8 @@ paths:
 
   /source/{project_name}/{package_name}:
     $ref: 'paths/source_project_name_package_name.yaml'
+  /source/{project_name}/{package_name}/_history:
```

Comment:
@eduardoj sometime I'm not sure if that really needs to be documented as a seperate endpoint. In theory this path is just  `/source/{project_name}/{package_name}/{file_name}` and `_history` is the file to receive. I know its kind of a special file, but also a bit inconsistent in terms of documenting endpoints.

Reply:
okay, I see, its a different endpoint on backend level https://github.com/openSUSE/open-build-service/blob/a3bd82a38dfba6ebef167057bf4da8868cab9a94/src/backend/bs_srcserver#L7640